orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt

```diff
@@ -59,15 +64,41 @@ class RedisActiveExecutionsMonitor(
 
   private val snapshot: MutableMap<Id, AtomicLong> = ConcurrentHashMap()
 
-  @Scheduled(fixedRateString = "\${queue.monitor.activeExecutions.register.frequency.ms:60000}")
-  fun registerGauges() {
+  private val executor = Executors.newScheduledThreadPool(2)
+
+  private val activePipelineCounter = registry.gauge(
+    registry.createId("executions.active").withTag("executionType", Execution.ExecutionType.PIPELINE.toString()),
+    AtomicInteger(0)
+  )
+
+  private val activeOrchestrationCounter = registry.gauge(
+    registry.createId("executions.active").withTag("executionType", Execution.ExecutionType.ORCHESTRATION.toString()),
+    AtomicInteger(0)
+  )
+
+  init {
+    executor.scheduleWithFixedDelay(
+      { refreshGauges() },
```

Comment:
What happens if this call throws an exception?

Reply:
Good call ... I've now wrapped in a try / catch to prevent the exception from (a) being lost and (b) preventing reschedule.